### ISSUE TITLE
Slightly misleading proxy error message

### ISSUE THREAD

**Issue by bboreham** [weaveworks/weave#1880]
````
$ weave launch-proxy
13c0f5c12f81392b2dc5ab437bc2092a2d72f2c8dd15630666fd66cb0cfb9451
$ weave launch-router --foo
The weave container has died. Consult the container logs for further details.
$ docker logs weaveproxy
[...]
INFO: 2016/01/21 14:06:58.981547 Attaching weave router container: 65f8dd12bee0ef2998c110f6597d89519cf972641544387f56ba0e03ec40210c
WARN: 2016/01/21 14:06:59.275428 Attaching container 65f8dd12bee0ef2998c110f6597d89519cf972641544387f56ba0e03ec40210c to weave network failed: The weave container has died. Consult the container logs for further details.

WARN: 2016/01/21 14:06:59.275448 unable to attach new container 65f8dd12bee0ef2998c110f6597d89519cf972641544387f56ba0e03ec40210c since inspecting it failed: The weave container has died. Consult the container logs for further details.
````

The part about "since inspecting it failed" is incorrect: the value of `err` may come from an `inspect` or it may come from the `attach`

**Comment by bboreham** [comment]
Also the print-out of the message is duplicated

**Comment by rade** [comment]
I am not seeing this.
```
$ weave reset
$ weave launch-proxy
c3809b8b53381c2103d842d6018b6206493355100ea9e3689eb5f10ec5d52b39
$ weave launch-router --foo
The weave container has died. Consult the container logs for further details.
$ docker logs weaveproxy
INFO: 2016/01/21 14:18:52.969217 weave proxy git-0872623ca56a
INFO: 2016/01/21 14:18:52.969265 Command line arguments: -H tcp://0.0.0.0:12375
INFO: 2016/01/21 14:18:52.970793 Docker API on unix:///var/run/docker.sock: &[Arch=amd64 KernelVersion=3.19.0-47-generic BuildTime=Fri Nov 20 13:12:04 UTC 2015 Version=1.9.1 ApiVersion=1.21 GitCommit=a34a1d5 GoVersion=go1.4.2 Os=linux]
INFO: 2016/01/21 14:18:53.052001 proxy listening on tcp://0.0.0.0:12375
INFO: 2016/01/21 14:19:04.929816 Attaching weave router container: b94c32ccfb6e1e7c289b21d25c60d4230ec6795802aee110171b1df15b04816e
WARN: 2016/01/21 14:19:05.395624 Attaching container b94c32ccfb6e1e7c289b21d25c60d4230ec6795802aee110171b1df15b04816e to weave network failed: The weave container has died. Consult the container logs for further details.
```

**Comment by rade** [comment]
Turns out that this got fixed by #1561.
